src/components/LandingPage/Footer/Footer.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import "bootstrap/dist/css/bootstrap.min.css";
```

Comment:
This import doesn't work, and please try to stick to Material UI. Could you please fix this?